src/asm2cfg/asm2cfg.py

```diff
@@ -119,8 +124,8 @@ def get_call_pattern(stripped):
     0x000055555557259c <+11340>:	addr32 call 0x55555558add0 <_Z19exportDebugifyStats>
     """
     if stripped:
-        return re.compile(r'0x0*([01-9a-fA-F]+):.*call\s*(.*[01-9a-fA-F]+.*)$')
-    return re.compile(r'<\+(\d+)>:.*call\s*(.*[01-9a-fA-F]+.*)$')
+        return re.compile(fr'0x0*({hex_pattern}):.*callq?\s*(.*{hex_pattern}.*)$')
+    return re.compile(fr'<[+-](\d+)>:.*callq?\s*(.*{hex_pattern}.*)$')
```

Comment:
Would you happen to have some example assembly file with AT&T syntax that could test this call pattern? When testing this PR I noticed this repo really needs a test suite.

Reply:
Ok, I'll add sample to examples.